chalicelib/services/InsightsService.py

```diff
@@ -70,19 +70,33 @@ def _get_dashboard_insights(data):
                 else:
                     majors[major] = 1
         
-        avg_gpa /= count_gpa
+        if count_gpa:
+            avg_gpa /= count_gpa
+        else:
+            avg_gpa = "N/A"
```

Comment:
I think using `None` is a better choice instead of using strings – having multiple datatypes being returned isn't too good of a practice